robot/src/robot.py

```diff
@@ -6,6 +6,7 @@
 import uasyncio
 
```

Comment:
And also import event (hopefully this import works):
```suggestion
from event import Event
```